packages/frontend/src/components/profile/authorized_apps/AuthorizedApp.js

```diff
@@ -59,6 +59,8 @@ const Container = styled.div`
 `;
 
 const AuthorizedApp = ({ app, onClick, deAuthorizing }) => {
+    console.log('app', app);
```

Comment:
Is the console.log intended here or shall we remove it?

Reply:
it is unintended, thanks for reviewing